src/assets/translations/de/main.json

```diff
@@ -1826,12 +1826,11 @@
       "description": "Wählen Sie aus der Liste unten die Konten aus, die Sie importieren möchten.",
       "import": "Importieren",
       "loadMoreDisabled": "Diese Wallet unterstützt keine Mehrfachkonten für diese Kette."
-    },
-    "ledgerOpenApp": {
-      "title": "Öffnen Sie die %{appName} App",
-      "description": "Um diese Kette zu verbinden, müssen Sie die Ledger-App auf Ihrem Gerät öffnen."
```

Comment:
q: I don't get why the description has been removed, we need to keep it in every translation files

Reply:
+1, it's consumed in `src/components/Modals/LedgerOpenApp/LedgerOpenAppModal.tsx`.

To revert this change.